api/tests/routes/backoffice/gdpr_user_extract_data_test.py

```diff
@@ -186,3 +188,43 @@ def test_no_file_in_bucket(self, authenticated_client):
         assert "L'extraction demandée existe mais aucune archive ne lui est associée" in html_parser.extract_alerts(
             redirection.data
         )
+
+
+class DeleteGdprUserExtractTest(PostEndpointHelper, StorageFolderManager):
+    endpoint = "backoffice_web.gdpr_extract.delete_gdpr_user_data_extract"
+    endpoint_kwargs = {"gdpr_id": 1}
+    needed_permission = perm_models.Permissions.EXTRACT_PUBLIC_ACCOUNT
+    storage_folder = settings.LOCAL_STORAGE_DIR / settings.GCP_GDPR_EXTRACT_BUCKET / settings.GCP_GDPR_EXTRACT_FOLDER
+
+    def test_delete_gdpr_user_extract(self, authenticated_client):
+        extract = users_factories.GdprUserDataExtractBeneficiaryFactory(dateProcessed=datetime.datetime.utcnow())
+        with open(self.storage_folder / f"{extract.id}.zip", "wb") as fp:
+            fp.write(b"[personal data compressed with deflate]")
+
+        response = self.post_to_endpoint(authenticated_client, gdpr_id=extract.id)
+        assert response.status_code == 302
+
+        expected_url = url_for("backoffice_web.gdpr_extract.list_gdpr_user_data_extract", _external=True)
+        assert response.location == expected_url
+
+        assert users_models.GdprUserDataExtract.query.count() == 0
+        assert len(os.listdir(self.storage_folder)) == 0
+
+    def test_delete_gdpr_user_extract_still_processed(self, authenticated_client):
```

Comment:
Plutôt not processed, non ?

Reply:
non c'est bien still processed car l'offre ne doit pas être en ready, le but étant de simuler l'erreur 403 si l'utilisateur tente de supprimer un extrait toujours en cours de traitement et non pret.

Reply:
« Processed » signifie traité. C'est contradictoire avec ta réponse.
Donc en cours de traitement serait « processing » !

Reply:
en effet, je vais mettre still_processing du coup :)